### PR TITLE
New Crowdin updates

### PR DESCRIPTION
**Pull request by jjw24**

### CODE REVIEW

File: Flow.Launcher/Languages/fr.xaml [hunk 1]
```diff
@@ -85,25 +85,25 @@
...
     <system:String x:Key="newPriority">Nouvelle priorité</system:String>
     <system:String x:Key="priority">Priorité</system:String>
-    <system:String x:Key="priorityToolTip">Change Plugin Results Priority</system:String>
+    <system:String x:Key="priorityToolTip">Changer la priorité des résultats du plugin</system:String>
     <system:String x:Key="pluginDirectory">Répertoire</system:String>
     <system:String x:Key="author">par</system:String>
     <system:String x:Key="plugin_init_time">Chargement :</system:String>
     <system:String x:Key="plugin_query_time">Utilisation :</system:String>
-    <system:String x:Key="plugin_query_version">Version</system:String>
+    <system:String x:Key="plugin_query_version">| Version</system:String>
```
Comment (jjw24) [inline Flow.Launcher/Languages/fr.xaml:106]: hmm this is intended?

File: Flow.Launcher/Languages/fr.xaml [hunk 2]
```diff
@@ -208,86 +208,86 @@
...
     <system:String x:Key="proxyIsCorrect">Le proxy est valide</system:String>
     <system:String x:Key="proxyConnectFailed">Connexion au proxy échouée</system:String>
 
     <!--  Setting About  -->
     <system:String x:Key="about">À propos</system:String>
-    <system:String x:Key="website">Site Web</system:String>
-    <system:String x:Key="github">GitHub</system:String>
-    <system:String x:Key="docs">Docs</system:String>
+    <system:String x:Key="website">Site web</system:String>
+    <system:String x:Key="github">Github</system:String>
```
Comment (jjw24) [inline Flow.Launcher/Languages/fr.xaml:220]: why would this be lowercased?

File: Flow.Launcher/Languages/fr.xaml (hunk 1 above)
Comment (VictoriousRaptor) [reply Flow.Launcher/Languages/fr.xaml:107]: IIRC it's from some old version

Comment (jjw24) [reply Flow.Launcher/Languages/fr.xaml]: Line removed via Crowdin

File: Flow.Launcher/Languages/fr.xaml (hunk 2 above)
Comment (jjw24) [reply Flow.Launcher/Languages/fr.xaml:221]: Ignoring

**Review by onesounds — APPROVED** [review]

**Pull request merged**